src/main/java/com/openclassroomsproject/paymybuddy/configuration/security/SecurityService.java

```diff
@@ -0,0 +1,32 @@
+package com.openclassroomsproject.paymybuddy.configuration.security;
+
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.server.VaadinServletRequest;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.web.authentication.logout.SecurityContextLogoutHandler;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SecurityService {
```

Comment:
I don't like the Service name, I would prefer to put sth like `SecurityProvider` but it's more a pov than an advice :)

Reply:
done, name changed.